Makefile

```diff
@@ -18,6 +18,7 @@ REDIS_PROFILE = redis
 RABBITMQ_PROFILE = rabbitmq
 PROMETHEUS_PROFILE = prometheus
 GRAFANA_PROFILE = grafana
+TELEGRAM_POLLING_PROFILE = telegram_polling
```

Comment:
👋 do you mind adding this new compose profile [here](https://github.com/grafana/oncall/blob/dev/dev/README.md#compose_profiles:~:text=The%20possible%20profiles%20values%20are%3A)?